### PR TITLE
[ci/release] Migrate golden notebook tests

### PR DESCRIPTION
**Pull request by krfricke**
## Why are these changes needed?

Migrating golden notebook tests to new release test package.

https://buildkite.com/ray-project/release-tests-branch/builds/132

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/golden_notebook_tests/golden_notebook_tests.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@
   run:
     use_connect: True
     autosuspend_mins: 10
-    timeout: 1200
+    timeout: 1500
```
Comment (matthewdeng) [inline release/golden_notebook_tests/golden_notebook_tests.yaml:10]: Is this file still needed if this is in `release_tests.yaml`?

File: release/ray_release/schema.json [hunk 2]
```diff
@@ -18,6 +18,9 @@
 				"legacy": {
 					"$ref": "#/definitions/Legacy"
 				},
+                "stable": {
+                    "type": "boolean"
+                },
```
Comment (matthewdeng) [inline release/ray_release/schema.json:23]: Indentation looks off here?

Comment (matthewdeng) [inline release/ray_release/schema.json:23]: Indentation looks off here?

**Review by matthewdeng — APPROVED** [review]

File: release/golden_notebook_tests/golden_notebook_tests.yaml (hunk 1 above)
Comment (krfricke) [reply release/golden_notebook_tests/golden_notebook_tests.yaml:10]: Good point, I'll remove it - thanks!

**Comment by krfricke** [comment]
Tests are passing: https://buildkite.com/ray-project/release-tests-branch/builds/155
